netpyne/sim/run.py

```diff
@@ -100,7 +100,7 @@ def recordLFPHandler():
 #------------------------------------------------------------------------------
 # Run Simulation
 #------------------------------------------------------------------------------
-def runSim ():
+def runSim (reRun=False):
```

Comment:
@blakecaldwell this looks like a very useful feature, thx! do you mind renaming this to skipPreRun ? I think reRun makes it sound like it's gonna run multiple times.